content/en/docs/reference/node/kubelet-files.md

```diff
@@ -0,0 +1,142 @@
+---
+content_type: "reference"
+title: Local Files And Paths Used By The Kubelet
+weight: 42
+---
+
+The {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} is mostly a stateless
+process running on a Kubernetes {{< glossary_tooltip text="node" term_id="node" >}}.
+This document outlines files that kubelet reads and writes.
```

Comment:
We should probably have a strong disclaimer about these implementation details not necessarily being stable / guaranteed?

Reply:
added a note below, good callout!